scripts/test-iaccount-execute.py

```
@@ -0,0 +1,497 @@
+    for ep in [ENTRYPOINT_V07, ENTRYPOINT_V08, ENTRYPOINT_V09]:
+        rpc("eth_sendTransaction", [{
+            "from": ANVIL_FUNDER,
+            "to": ep,
+            "data": "0xb760faf9" + padded,
+            "value": "0x8AC7230489E80000",
+            "gas": "0x30000",
+        }])
+
+    rpc("anvil_mine", ["0x4"])
+    print(f"  Account deployed at {ACCOUNT_ADDRESS}")
+    print(f"  Deposited 10 ETH into each EntryPoint for account")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)